docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -21,16 +21,17 @@ The default Helm chart deploys the following components:
 - **QueryFrontend component** (2 replicas, maxUnavailable: 1): Manages frontend queries. Up to 1 replica can be unavailable during updates.
 - **QueryScheduler component** (2 replicas): Schedules queries.
 
-It is not recommended to run scalable mode with `filesystem` storage. For the purpose of this guide, we will use MinIO as the object storage to provide a complete example. 
+{{< admonition type="note" >}}
```

Comment:
this doesn't necessarily require a change, but I'm curious why this is the right spot to make this point.

I understand it's important and why it's here, but an alternative could be to guide me down the path of the opinionated right way of doing things, and simply provide a summary block somewhere else saying "we chose to embed the following best practices in this approach"

Reply:
Good point! I took it as we need to tell people quickly not to choose filesystem as there default. Its an option and the easiest to deploy but not recommended for any deployment other than single node. Will see if I can make this less blunt